### ISSUE TITLE
Cannot use @wdio/selenium-standalone-service with Edgedriver, either with binary or version keys, or with javaArgs

### ISSUE THREAD

**Issue by osmolyar** [webdriverio/webdriverio#5392]
**Environment**:

    WebdriverIO version: 6.0.15
    Mode: selenium-standalone-service
    Node.js version: 12.9.0
    NPM version: 6.14.4
    Browser name and version: Any (Firefox 75.0)
    Platform name and version:. Windows 10
    Additional wdio packages used (if applicable)::
    "@wdio/allure-reporter": "^6.1.9",
    "@wdio/cli": "^6.0.15",
    "@wdio/cucumber-framework": "^6.0.15",
    "@wdio/dot-reporter": "^6.0.14",
    "@wdio/junit-reporter": "^6.0.15",
    "@wdio/local-runner": "^6.0.15",
    "@wdio/selenium-standalone-service": "^6.0.15",
    "@wdio/spec-reporter": "^6.0.15",
    "@wdio/sync": "^6.0.15",
    "wdio-cucumberjs-json-reporter": "^1.2.4",
    "webdriverio": "^6.0.15",

**Config of WebdriverIO**
```js
services: [ 
        ['selenium-standalone', {
            logPath: 'logs',
            installArgs: {
                drivers: {
                    chrome: { version: '81.0.4044.69' },
                    firefox: { version: '0.26.0' },
                    // edge: {version: '81.0.409.0'}, //doesn't install correct version and doesn't let you sub in correct driver renamed as grid does
                    ie: {
                        version: "3.5.1",
                        arch: process.arch,
                        baseURL: "https://selenium-release.storage.googleapis.com"
                    },
                    edge: {
                        binary: path.join(__dirname, 'msedgedriver.exe') // has to be absolute
                        // binary: "C:\drivers\msedgedriver.exe"
                    }
                }
            },
            args: {
                drivers: {
                    chrome: { version: '81.0.4044.69' },
                    firefox: { version: '0.26.0' },
                    // edge: {version: '81.0.409.0'} //doesn't install correct version and doesn't let you sub in correct driver renamed as grid does
                    ie: {
                        version: "3.5.1",
                        arch: process.arch,
                        baseURL: "https://selenium-release.storage.googleapis.com"
                    },
                    edge: {
                        binary: path.join(__dirname, 'msedgedriver.exe') // has to be absolute
                        // binary: "C:\drivers\msedgedriver.exe"
                    }
                },
             // javaArgs: [
               //         '-Dwebdriver.edge.driver="C:\\Windows\\System32\\MicrosoftWebDriver.exe"'
        //     ]
            },
        }]
    ],
```

**Describe the bug**
Firstly, and this is not a webdriverio issue, selenium-standalone does not download the specified version of Edgedriver, instead always installing the obsolete 17134-MicrosoftEdgeDriver.exe version for the ie-based edge, so the drivers.edge.version setting cannot be used.  

The suggested workarounds are either to use a drivers.edge.binary=<path to binary> or, according to selenium-standalone-service documentation, which may or may not be current for wdio v6, to use a javaArgs key passing the -Dwebdriver.edge.driver parameter.  Neither approach works; if edge is included in the installArgs, either with binary or version setting, it tries to install and use the _17324 version regardless. If it's included in the args with the binary setting, without including it in installArgs, it errors out saying the driver is missing (driver.version unknown).  

Also if a test is run once with 'edge' uncommented in the capabilities and args, no further tests with any drivers will work, even if edge is commented out everywhere, saying that the service could not be started and to make sure it's running on the port, first complaining that the 17134-MicrosoftEdgeDriver.exe cannot be written to.  This binary also cannot be deleted manually, saying it's in use by webdriver.  The MicrosoftWebDriver processes must be killed via Task Manager  in order to be able to use the service with other drivers.

**To Reproduce**
Steps to reproduce the behavior:  As described above

**Expected behavior**
Expect either drivers.edge.binary: or javaArgs: to override the automatic 17134-MicrosoftEdgeDriver.exe downloading behavior to be able to use a manually downloaded MSEdgedriver binary.

**Log**

**Additional context**

**Comment by christian-bromann** [comment]
> selenium-standalone does not download the specified version of Edgedriver,

@osmolyar would you mind filling an issue in the [`selenium-standalone`](https://github.com/vvo/selenium-standalone) package for this?

**Comment by osmolyar** [comment]
Submitted https://github.com/vvo/selenium-standalone/issues/458

**Comment by erwinheitzman** [comment]
You are assigning the driver binary though, which instead should be the browser binary. Have you tried that?

**Comment by christian-bromann** [comment]
Thanks for submitting, this has no been resolved in v6.20.0 of that dependency and therefor should be resolved for you once you re-install all dependencies. Thanks!
